docs/types.md

```diff
@@ -782,7 +782,7 @@ All types of FileSpecs share some common fields:
 
 ### FileSpec -- Path
 
-A FileSpec with `type` equal to `path` points to an absolute or relative path accessible from the current machine.
+A FileSpec with `type` equal to `path` points to an absolute or relative path accessible from the current machine. The absolute or relative path may also include a glob pattern to target multiple files or directories.
```

Comment:
Watch the first rule of code style: be consistent with the code around you.
We break lines by sentence (or clause for long sentences).
This allows us to have better diffs (e.g., if you changed one work in a huge paragraph, you would get a hard to read diff).